### ISSUE TITLE
3.3.8 Accessible Authentication (Minimum) (AA)

### ISSUE THREAD

**Issue by AlainGravelet** [w3c/wcag#3306]
Some companies like banks (in US and Canada), for security purpose, use personal questions coming from a third party data base for authentication. Questions like the following ones. Is this method compliant with the criteria?
Thank you.

Questions examples:
Which of the following addresses have you ever been associated with?
1686 Hollander Court
2643 Westchester Parkway Southeast
5346 Laurel Falls
635 Kennolia Drive Southwest
I have never been associated with any of these addresses

Which of the following professional licenses have you ever held?
Dental Hygienist
Home Healthcare - Retail Pharmacy
Orthopedic Trauma
Veterinary Faculty
None of the above

Which of the following vehicles have you recently owned or leased?
1984 Bmw 3 Series
1986 Chevrolet El Camino
1987 Buick Electra
1993 Volkswagen Eurovan
I have never been associated with any of these vehicles

Which of the following colleges have you attended?
Cleveland Community College
Harcum College
Mitchell Technical Institute
Vanguard University Of Southern California
None of the above

In which of the following cities does 'Rolando Harkenreadeo' currently live or own property?
Carl Junction, Missouri
Chillicothe, Missouri
Florissant, Missouri
Troy, Missouri
None of the above or I am not familiar with this person

**Comment by alastc** [comment]
This method is a cognitive funtion test (remembering things), so _without an alternative method_ it would fail.

That is not to say they cannot include that method, but there must be another method that does meet the guideline.

**Comment by alastc** [comment]
No reply received, so closing as answered.
